app/src/views/disk-view.js

```diff
@@ -46,7 +47,9 @@ export default class DiskView {
   }
 
   _handleChanges(changes) {
-    if (changes.hasOwnProperty('disk') && changes.disk.hasOwnProperty('level') || changes.currentGame === "disk") {
+    if (changes.currentGame === GAMES.HANDSHAKE || // Reset on startup
```

Comment:
Well this is a very view specific requirement. I don't think there is an initial state change to put here. I suggest adding a public `reset()` method to all of the views and call that in start first game